accelerator/core/apis.c

```diff
@@ -394,6 +394,27 @@ status_t api_fetch_txn_with_uuid(const ta_cache_t* const cache, const char* cons
   return ret;
 }
 
+status_t api_register_mam_channel(const ta_cache_t* const cache, const char* const obj, char** json_result) {
+  status_t ret = SC_OK;
+  ta_register_mam_channel_req_t* req = ta_register_mam_channel_req_new();
+
+  ret = register_mam_channel_req_deserialize(obj, req);
+  if (ret) {
+    ta_log_error("%s\n", ta_error_to_string(ret));
```

Comment:
Add verbose messages.

Reply:
But this is the ordinary way we print logging message in TA

Reply:
No, you shall write the messages in more informative ways. i.e., explain why MAM channel was failing to be registered.